### PR TITLE
Completed Project

### PR DESCRIPTION
**Pull request by daashandayo**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+.input-container label {
+    display: block;
+
+}
+
+.submit {
+    background-color: #0077C5;
+    border: 1px solid #0077C5;
```
Comment (zlayabekrija) [inline main.css]: Border color and background color are the same. What is the need of border then?

File: main.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+.container {
+    width: 25em;
+    margin: 0 auto;
+}
+
+#header-title {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: space-evenly;
```
Comment (zlayabekrija) [inline main.css]: Please remove this, doesn't have any purpose.

File: main.css (hunk 1 above)
Comment (daashandayo) [reply main.css]: I agree.

File: main.css (hunk 2 above)
Comment (daashandayo) [reply main.css]: Removed 
```css
align-items: space-evenly;
```

**Comment by daashandayo** [comment]
> Fix lines that I have commented. You have my thumbs up. No need for further review.
> 👍
> Happy codin. :)

Thank You :)

**Pull request merged**